src/Mvc/Mvc.Testing/src/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -25,28 +30,23 @@
 
   <Target Name="_AddContentRootForProjectReferences" BeforeTargets="GetAssemblyAttributes" DependsOnTargets="_ResolveMvcTestProjectReferences">
```

Comment:
I think we can wire it up to a run as part of a different target if we'd like to. Maybe `PrepareForRun` or `PrepareResources`?